config/initializers/session_store.rb

```diff
@@ -1,6 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-Rails.application.config.session_store :cookie_store,
```

Comment:
so looking at the documentation/source code it appears the proper way to achieve no session 
is by passing `:disabled`